miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/context/WebConfig.java

```diff
@@ -1,5 +1,12 @@
 package uk.ac.bbsrc.tgac.miso.webapp.context;
 
+import java.util.HashMap;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.HashMap'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)